frontend/src/components/Transactions/CustomImporters/BaseCSVImporter.ts

```diff
@@ -16,31 +16,38 @@ type LoggerFn = (i: string, msg: string) => void;
 
 export interface CsvDataInterface {
   currency: string;
-  date: string;
-  price: string;
-  quantity: string;
-  fees: string;
-  totalExcludingFees: string;
+  date: dayjs.Dayjs;
+  price: number|null;
+  quantity: number|null;
+  fees: number|null;
+  totalExcludingFees: number|null;
```

Comment:
I used this interface in CSVExporter and these changes will probably break that code. Number type should work with a few changes, but Dayjs will be more difficult i suppose.

Reply:
Yep, it'll definitely break it 😎 !

I'll update the exporter too, don't worry!